src/app/(main)/blog/[slug]/page.tsx

```
@@ -0,0 +1,111 @@
+                {post.title}
+              </h1>
+            </div>
+
+            <div
+              className="blog-post-content"
+              style={{
+                fontSize: "12px",
+                lineHeight: "1.6",
+                color: "var(--win98-text, black)",
+              }}
+              dangerouslySetInnerHTML={{ __html: post.content }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)